src/components/Form/Form.js

```diff
@@ -116,7 +117,7 @@ export function Form({ children, config }) {
           fieldConfig.validators
             .reduce((errors, validator) => errors.concat(validator(field)), [])
             .filter((x) => !!x) // remove empty strings
-            .join(', '),
+            .join('. '),
```

Comment:
I've  removed the double error messaging  we saw on Zip, but, in the event we do have multiple, these should be sentances e.g. "Please do this. Please do that."